contracts/AddressRegistry.sol

```diff
@@ -51,4 +53,36 @@ contract AddressRegistry is Ownable {
         addressList.removeAddress(_previousContract, _address); 
         emit AddressRemoved(_address);
     } 
+
+    /// @notice Removes every address from the list
+    function clearAll() public onlyOwner {
```

Comment:
@asselstine Should we add an "AllAddressesRemoved" event for this?

Reply:
Makes sense to me!  Need events on any major state change